examples/react-hmr/webpack.config.js

```diff
@@ -2,8 +2,8 @@ const WebExtension = require('webpack-target-webextension')
 const webpack = require('webpack')
 const { join } = require('path')
 const ReactRefreshTypeScript = require('react-refresh-typescript')
-const RefreshPlugin = require('@pmmmwh/react-refresh-webpack-plugin')
```

Comment:
Unused

Reply:
Actually it is used

Reply:
`RefreshPlugin` does not appear anywhere else in this file

Reply:
oh I found I import this plugin twice